admin_bot/cogs/tools.py

```diff
@@ -67,35 +71,17 @@ async def feature(self, ctx, *args):
         )
 
     @commands.command()
-    async def get_help(self, ctx, team, *args):
+    @commands.has_any_role("Lead of Software", "Assistant lead of software")
+    async def kill(self, ctx):
```

Comment:
The get help command becomes the kill command? :o

Reply:
hm. I'll fix that